hotness/tests/test_helpers.py

```diff
@@ -6,10 +6,70 @@
 import re
 import unittest
 
+from unittest.mock import patch
 from hotness import helpers
 
 
 class TestHelpers(unittest.TestCase):
+    def test_expand_subdirs(self):
```

Comment:
Could you split this into multiple tests? It's good to have one test case = one test.

Reply:
Sure thing. How about now?